### ISSUE TITLE
CI is often failing with InteractiveHost32.exe" not found error.

### ISSUE THREAD

**Issue by AlekseyTs** [dotnet/roslyn#46532]
```
F:\workspace\_work\1\s\.tools\msbuild\16.6.5-alpha1\tools\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "F:\workspace\_work\1\s\artifacts\bin\InteractiveHost32\Release\net472\InteractiveHost32.exe.config" because it was not found. [F:\workspace\_work\1\s\src\Interactive\HostTest\InteractiveHost.UnitTests.csproj]
##[error].tools\msbuild\16.6.5-alpha1\tools\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "F:\workspace\_work\1\s\artifacts\bin\InteractiveHost32\Release\net472\InteractiveHost32.exe.config" because it was not found.
F:\workspace\_work\1\s\.tools\msbuild\16.6.5-alpha1\tools\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "F:\workspace\_work\1\s\artifacts\bin\InteractiveHost32\Release\net472\InteractiveHost32.exe" because it was not found. [F:\workspace\_work\1\s\src\Interactive\HostTest\InteractiveHost.UnitTests.csproj]
##[error].tools\msbuild\16.6.5-alpha1\tools\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "F:\workspace\_work\1\s\artifacts\bin\InteractiveHost32\Release\net472\InteractiveHost32.exe" because it was not found.
```

https://dev.azure.com/dnceng/public/_build/results?buildId=756187&view=logs&j=ccc4f4cd-3db2-535a-b197-b0dc972cf7f6&t=b5751933-4a09-5014-1b76-81e74adfd51c

**Comment by AlekseyTs** [comment]
CC @tmat

**Comment by AlekseyTs** [comment]
CC @dotnet/roslyn-infrastructure

**Comment by jinujoseph** [comment]
Fixed by #46211
